browser/components/loop/content/css/panel.css

```diff
@@ -254,29 +278,23 @@ html[dir="rtl"] .tab-view li:nth-child(2).selected ~ .slide-bar {
 /* Rooms and contacts shared CSS */
 
 .contact-list-empty,
-.room-list-empty {
-  background-image: url("../shared/img/empty_contacts.svg");
+.contact-search-list-empty,
+.no-conversations-message {
   background-repeat: no-repeat;
   background-position: top center;
-  padding-top: 19%;
-  padding-bottom: 3%;
   text-align: center;
   color: #4a4a4a;
   font-weight: lighter;
 }
 
-/* Don't show the empty contacts image if we're showing gravatar promo. */
-.contacts-gravatar-promo ~ .contact-list-empty {
-  background-image: none;
-  padding-top: 3%;
-}
 
-.contact-list-empty {
-  padding-top: 27%;
-}
 
-.room-list-empty {
-  margin: 5% 15px;
+.no-conversations-message {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  padding-top: 75px;
+  padding-bottom: 10px;
```

Comment:
Please add a comment here, describing what's happening, perhaps even linking to the stackoverflow reference.